docs/FRSKY_R9.md

```diff
@@ -236,38 +236,38 @@ Once the ELRS bootloader is installed, the following precedure can be followed t
 
 4. Select the FIRMWARE folder and scroll to and select the tx-R9M-f103c8-elrs-bl-v*.elrs file. Choose "Flash external ELRS".
 
-### Flash R9 receivers with ELRS bootloader ###
+### Flash R9 Receivers with ELRS Bootloader ###
 
-#### Flash the ELRS bootloader ####
+#### Flash the ELRS Bootloader ####
 
-Wiring the receiver to the JR Bay on the back of your radio and flashing the ELRS bootloader is documented [here](https://www.expresslrs.org/quick-start/receivers/r9/). The procedure is similar to flashing the R9M as described above in the First time bootloader install section. Use the appropriate .frk file indicated in the ELRS instructions for the receiver you are flashing. Flashing the ELRS bootloader from the .frk file only needs to be done once.
+Wiring the receiver to the JR Bay on the back of your radio and flashing the ELRS bootloader is documented [here](https://www.expresslrs.org/quick-start/receivers/r9/). The procedure is similar to flashing the R9M module as described above in the bootloader install section. Use the appropriate .frk file indicated in the ELRS instructions for the receiver you are flashing. Flashing the ELRS bootloader from the .frk file only needs to be done once.
 
-#### Flash/Update the mLRS firmware ####
+#### Flash/Update the mLRS Firmware ####
 
-After flashing the ELRS bootloader, you can connect the serial port as described in the link above and use the ELRS UARTupload.py script to flash/update mLRS with latest mLRS firmware .elrs file. There are, of course, separate .elrs image files for each receiver:
+After flashing the ELRS bootloader, you can connect the receiver's serial port as described in the link above and use the ELRS UARTupload.py script to flash/update with latest mLRS firmware .elrs file. There are, of course, separate .elrs image files for each receiver.
```

Comment:
Small rewrite (the instead of with):

After flashing the ELRS bootloader, you can connect the receiver's serial port as described in the link above and use the ELRS UARTupload.py script to flash/update the latest mLRS firmware .elrs file.'

Reply:
done